docs/source/cpp/compute.rst

```diff
@@ -455,20 +455,24 @@ Structural transforms
 +--------------------------+------------+------------------------------------------------+---------------------+---------+
 | is_null                  | Unary      | Any                                            | Boolean             | \(2)    |
 +--------------------------+------------+------------------------------------------------+---------------------+---------+
-| is_valid                 | Unary      | Any                                            | Boolean             | \(2)    |
+| is_valid                 | Unary      | Any                                            | Boolean             | \(3)    |
 +--------------------------+------------+------------------------------------------------+---------------------+---------+
-| list_value_length        | Unary      | List-like                                      | Int32 or Int64      | \(4)    |
+| is_nan                   | Unary      | Float, Double                                  | Boolean             | \(4)    |
```

Comment:
nit: move `is_nan` above `is_null` to keep names ordered

Reply:
Done.